metrics/cli/test/services/sentry/adapter/sentry_cli_service_adapter_test.dart

```diff
@@ -111,35 +111,32 @@ void main() {
     );
 
     test(
-      ".createRelease() prompts the user to enter the organisation slug",
+      ".getSentryRelease() prompts the user to enter the organisation slug",
       () async {
         whenPromptReleaseName().thenReturn(releaseName);
 
-        await sentryService.createRelease([sourceMap]);
+        sentryService.getSentryRelease();
 
         verify(prompter.prompt(enterOrganizationSlug)).called(once);
       },
     );
 
     test(
-      ".createRelease() throws if prompter throws during the organisation slug prompting",
+      ".getSentryRelease() throws if prompter throws during the organisation slug prompting",
```

Comment:
```suggestion
      ".getSentryRelease() throws if prompter throws during the organization slug prompting",
```
🏷️  Please, apply this suggestion to similar cases in this file.